minecraft_servers/theflare/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+  "server_name": "theflare",
+  "nice_name": "theFLARE",
+  "direct_ip": "trueflare.net",
+  "supported_languages": [
+    "en"
+  ],
+  "social": {
+    "web": "https://trueflare.net",
```

Comment:
Is your website down? 
`Verbindung fehlgeschlagen: Access denied for user 'JamesBond'@'localhost' (using password: YES)`